utils/validateUser.js

```diff
@@ -3,13 +3,13 @@ const Joi = require('@hapi/joi');
 exports.validateRegisterInput = Joi.object({
```

Comment:
Please, rename the file to `validate-user`

Reply:
fixed